src/main/java/com/washingtonpost/arc/ans/v0_3/model/ANS.java

```diff
@@ -22,7 +22,7 @@
     @Type(value = Media.class, name = "media"),
     @Type(value = Image.class, name = "image"),
     @Type(value = Video.class, name = "video"),
-    @Type(value = Story.class, name = "story"),
+    @Type(value = Collection.class, name = "story"),
```

Comment:
And I suppose it's sort of a TBD if we would want someone to declare a block of json as a type="collection" ... we do allow a type="media" which is kind of the parallel concept over in the "video & image" side of the schema.  So if you think you might want a concrete Collection instance at some point, go ahead and add a Type annotation that maps Collection.class to name "collection"